ci/scripts/filter/filter.go

```
@@ -94,6 +112,10 @@ nextline:
 			}
 		}
 
+		for _, r := range replacements {
+			line = r.replace(line)
+		}
```

How does this perform in practice, both for the retail-demo data and 6X ICW? The new replacement regex doesn't have any anchors to help the regex engine fail fast.

---

If I remember correctly, it was about ~17 seconds without anchoring, and ~13 seconds with.